jetstream/kv.go

```diff
@@ -611,25 +633,43 @@ func (js *jetStream) prepareKeyValueConfig(ctx context.Context, cfg KeyValueConf
 	if cfg.Compression {
 		compression = S2Compression
 	}
+	var allowMsgTTL bool
+	var subjectDeleteMarkerTTL time.Duration
+	if cfg.LimitMarkerTTL != 0 {
+		if cfg.LimitMarkerTTL < time.Second {
```

Comment:
Do we want to make a client side validation here?
Why not rely on the server side validation?

Reply:
Removed client side validation.